creator-node/src/routes/users.js

```diff
@@ -220,7 +223,36 @@ module.exports = function (app) {
         response.syncInProgress = syncInProgress
       }
 
-      await Promise.all([fetchClockValueFromDb(), isSyncInProgress()])
+      async function fetchFilesHashIfRequested() {
+        if (returnFilesHash && cnodeUser) {
+          const filesHash = await DBManager.fetchFilesHashFromDB({
+            lookupKey: { lookupCNodeUserUUID: cnodeUserUUID }
+          })
+          response.filesHash = filesHash
+
+          const filesHashClockRangeMin =
+            req.query.filesHashClockRangeMin || null
+          const filesHashClockRangeMax =
+            req.query.filesHashClockRangeMax || null
+
+          if (filesHashClockRangeMin || filesHashClockRangeMax) {
+            const filesHashForClockRange = await DBManager.fetchFilesHashFromDB(
+              {
+                lookupKey: { lookupCNodeUserUUID: cnodeUserUUID },
+                clockMin: filesHashClockRangeMin,
+                clockMax: filesHashClockRangeMax
+              }
+            )
+            response.filesHashForClockRange = filesHashForClockRange
```

Comment:
nit - lets keep the name of this prop consistent across routes

Reply:
the name is consistent across routes, `filesHash` is used everywhere to represent the hash for all files for user
see line 231 above in this same route
this prop is returned in addition to `filesHash` to represent filesHash for the reuqested clock range